packages/ai-mistral/tests/mistral-adapter.test.ts

```
@@ -1,17 +1,31 @@
-import {
-  describe,
-  it,
-  expect,
-  vi,
-  afterEach,
-  beforeEach,
-  type Mock,
-} from 'vitest'
-import { createMistralText, mistralText } from '../src/adapters/text'
-import { transformNullsToUndefined } from '../src/utils/schema-converter'
+import { describe, it, expect, vi, afterEach, beforeEach } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)